cmd/kubestellar-scheduler/cmd/scheduler.go

```diff
@@ -159,8 +163,11 @@ func Run(ctx context.Context, options *scheduleroptions.Options) error {
 		edgeClusterClientset,
 		edgeSharedInformerFactory.Edge().V1alpha1().EdgePlacements(),
 		edgeSharedInformerFactory.Edge().V1alpha1().SinglePlacementSlices(),
-		schedulingSharedInformerFactory.Scheduling().V1alpha1().Locations(),
-		workloadSharedInformerFactory.Workload().V1alpha1().SyncTargets(),
+		// should these next two lines be commented out? Or should we still allow kcp objects?
```

Comment:
Yes, commented out.  We want to stop using the object kind definitions found in the kcp repo.
You should be getting warnings that schedulingSharedInformerFactory and workloadSharedInformerFactory are not used; you can entirely remove them.